### PR TITLE
Implemented C# 9.0 partial method example

### PR DESCRIPTION
**Pull request by MarkMichaelis**

### CODE REVIEW

File: src/Chapter06/Listing06.53.DefiningAPartialMethodToAccessItsImplementation.cs [hunk 1]
```diff
@@ -56,18 +56,18 @@ public string FirstName
...
         static partial void OnLastNameChanging(string value)
         {
-            if(value is null)
-            {
+            value = value ?? 
                 throw new ArgumentNullException(nameof(value));
-            }
-            if(value.Trim().Length == 0)
+
+            if (value.Trim().Length == 0)
```
Comment (BenjaminMichaelis) [inline src/Chapter06/Listing06.53.DefiningAPartialMethodToAccessItsImplementation.cs]: Can we use string.IsNullOrWhitespace() to simplify this?

File: src/Chapter06/Listing06.53.DefiningAPartialMethodToAccessItsImplementation.cs [hunk 2]
```diff
@@ -77,10 +77,9 @@ partial class Person
         #region EXCLUDE
         static partial void OnFirstNameChanging(string value)
         {
-            if(value == null)
-            {
+            value = value ?? 
                 throw new ArgumentNullException(nameof(value));
-            }
+
             if (value.Trim().Length == 0)
```
Comment (BenjaminMichaelis) [inline src/Chapter06/Listing06.53.DefiningAPartialMethodToAccessItsImplementation.cs]: Same here

File: src/Chapter06/Listing06.53.DefiningAPartialMethodToAccessItsImplementation.cs [hunk 3]
```diff
@@ -90,6 +89,9 @@ partial class Person
             }
         }
         #endregion EXCLUDE
+
+
```
Comment (BenjaminMichaelis) [inline src/Chapter06/Listing06.53.DefiningAPartialMethodToAccessItsImplementation.cs]:
```suggestion
```
Nit: whitespace

**Review by BenjaminMichaelis — APPROVED** [review]

File: src/Chapter06/Listing06.53.DefiningAPartialMethodToAccessItsImplementation.cs [hunk 4]
```diff
@@ -25,7 +25,7 @@ public string LastName
             }
             set
             {
-                if((_LastName != value))
+                if ((_LastName != value))
```
Comment (Keboo) [inline src/Chapter06/Listing06.53.DefiningAPartialMethodToAccessItsImplementation.cs]: nit:
```suggestion
                if (_LastName != value)
```

**Review by Keboo — APPROVED** [review]

**Pull request merged**